backend/services/users.js

```diff
@@ -55,8 +55,13 @@ async function resetPassword(data) {
   await PasswordReset.deleteOne({ _id: password_reset._id }).exec();
 }
 
+async function getAllUsers() {
```

Comment:
Same thing here, just add a comment for consistency.

Reply:
done!